### DISCUSSION TITLE
Codeclimateによるリファクタリングの提案

### DISCUSSION THREAD

**Discussion by REI-MORI** [sinProject-Inc/talk#1448] [Q&A]
@iamsin 
Codeclimateより重複コードをリファクタリングするよう提案がありました。

https://codeclimate.com/github/sinProject-Inc/talk/pull/1446

以下の2つのファイルは処理内容がほとんど同じです。このクラスを作成された方は、役割が違うのでクラスを分けているのかと思いますのでそのままにしておいた方が良いと思います。2つのクラスを1つに統合すべきか、このCodeclimateのエラーは無視するべきかご意見いただけませんでしょうか。

src/lib/view/message.ts
src/lib/general/text_error.ts

**Comment by iamsin** [comment]
@REI-MORI 

ご連絡ありがとうございます。この指摘は、Wontfix を選択して無視しておいてくださいませ。

**Reply by REI-MORI** [reply]
@iamsin かしこまりました。ありがとうございます。

**Reply by REI-MORI** [reply]
@iamsin Wontfixにしたいのですが選択できない状態です。権限など関係ありますでしょうか。
![image](https://github.com/sinProject-Inc/talk/assets/93172801/e8da3e5a-e014-4641-85ed-f141aa6fe9d2)

**Reply by sinproject-iwasaki** [reply]
@REI-MORI 

権限をどこで設定するかが分からなかったため、こちらで対応しておきました。
この対応で Issue を作成していると思いますので、Issue 二関連する質問は Issue に今後はご記載ください。
@iamsin は予備アカウントです。メンションは @sinproject-iwasaki までお送りくださいませ。
